src/index.ts

```diff
@@ -6,6 +6,7 @@ const kc = new k8s.KubeConfig();
 kc.loadFromDefault();
 const k8sApi = kc.makeApiClient(k8s.CoreV1Api);
 const k8sContext = kc.getCurrentContext();
+const client = kc.makeApiClient(k8s.CustomObjectsApi);
```

Comment:
should we make this consistent with other k8s API object like this
https://github.com/open-services-group/peribolos-as-a-service/pull/41/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R8 ?

just for easy of readability

Reply:
Good point changed it. Thanks :+1: